website/src/components/CalendarIcon.tsx

```diff
@@ -0,0 +1,33 @@
+import type { ComponentPropsWithoutRef } from 'react'
+
+interface CalendarIconProps extends ComponentPropsWithoutRef<'svg'> {
+  value?: number
+}
+
+export const CalendarIcon = ({ value = 1 }: CalendarIconProps) => {
```

Comment:
What if the default was the current date? Haha

Reply:
i think its a pretty solid idea